index.html

```diff
@@ -4832,6 +4833,94 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="mark">
+			<rdef>mark</rdef>
+			<div class="role-description">
+				<p>Content which is marked or highlighted for reference or notation purposes, due to the content's relevance in the enclosing context.</p>
+				<p>Example uses for <rref>mark</rref> include:</p>
+				<ul>
+					<li>Highlighting text in a quotation which is of special interest but is not marked in the original source material, comparable to using a highlighter pen to mark passages of a print article.</li>
+					<li>Indicating portions of the content that are relevant to the user's current activity, such as highlighting text matches found by a search feature.</li>
+				</ul>
+				<p>Don't use <rref>mark</rref> for purely decorative styling such as syntax highlighting.</p>
```

Comment:
What abaout "Authors SHOULD  not use mark...."?

Nit: In addition, please use `<code>` tags instead of `<rref>` when referring to the thing being defined in the section. The reason why is the first instance of an `<rref>` in any section will be a link to the associated content. I personally don't think there's any reason why we want a link in the `mark` section to the `mark` section.

Reply:
Good suggestions. Done.